src/formatDate.js

```diff
@@ -50,7 +50,98 @@
  */
 
 function formatDate(date, fromFormat, toFormat) {
-  // write code here
+  const splitedDate = date.split(fromFormat[3]);
+  let year = '';
+  let month = '';
+  let day = '';
+
+  for (let i = 0; i < toFormat.length; i++) {
+    switch (fromFormat[i]) {
+      case 'YYYY':
+        year = splitedDate[i];
```

Comment:
instead of saving it to separate variables, create an object and save values under the same keys, e.g.
`{ YYYY: 2022, MM: 10, DD: 12 }`
in this case, in the next steps, you will know which format you have saved